AppCenterAnalytics/AppCenterAnalytics/Internals/Util/MSAnalyticsConstants.h

```diff
@@ -7,3 +7,13 @@
 static const int kMSLongMetadataTypeId = 4;
 static const int kMSDoubleMetadataTypeId = 6;
 static const int kMSDateTimeMetadataTypeId = 9;
+
+/**
+ * Minimum flush interval for channel.
+ */
+static NSUInteger const MSFlushIntervalMinimum = 3.0;
```

Comment:
Need need for a decimal portion since it's an integer

```suggestion
static NSUInteger const MSFlushIntervalMinimum = 3;
```

Reply:
Same for` MSFlushIntervalDefault` above. You also need the `k` prefix.
```suggestion
static NSUInteger const kMSFlushIntervalMinimum = 3;
```